### PR TITLE
Updated the navbar

### PR DESCRIPTION
**Pull request by uvsingh2706**
Screenshot 2022-09-30 at 5 06 05 PM
hey @Jai-Solania-29 , is it fine now

### CODE REVIEW

**Review by Kanika637 — APPROVED** [review]
LGTM🚀

File: src/Navbar.js [hunk 1]
```diff
@@ -21,24 +24,56 @@ function Navbar() {
...
+        Buy Again
+        </a>
+        <a class="navbar-brand white-text mr-auto" href="#" style={{fontSize:"10px",  fontWeight:"bold"}}>
+        Health, Household & Personal care
+        </a>
+        <a class="navbar-brand white-text mr-auto" href="#" style={{fontSize:"10px",  fontWeight:"bold"}}>
+        Baby
+        </a>
+        <a class="navbar-brand white-text mr-auto" href="#" style={{fontSize:"10px",  fontWeight:"bold"}}>
+        Toys& Games
```
Comment (Jai-Solania-29) [inline src/Navbar.js:60]:
```suggestion
        Toys & Games 
```

**Comment by Kanika637** [comment]
Great work @uvsingh2706 🚀🙌

**Review by Jai-Solania-29 — CHANGES_REQUESTED** [review]
Great work @uvsingh2706  🎉
Just make the minor changes that I have suggested, and then we are good to go 🚀

**Review by uvsingh2706 — COMMENTED** [review]
i have make the changes

**Comment by uvsingh2706** [comment]
hey @Jai-Solania-29, check it now i have make the changes

**Pull request merged**